WordPress/gutenberg
# Apply hover styles to filter chip while being configured

## What?
Apply hover styles to filter chip while the filter is being configured.

## Why?
Further helps connect the popover with the associated chip. This will become more important as more filters are added.

## How?
Target the `aria-expanded` attribute and style accordingly.

## Testing Instructions
1. Open a data view in the site editor e.g. "Manage all pages"
2. Add filters
3. Notice that while configuring a filter, the associated chip has a different appearance (darker background) to any other chips.


| Before | After |
| --- | --- |
| <img width="664" alt="Screenshot 2024-02-21 at 11 41 09" src="https://github.com/WordPress/gutenberg/assets/846565/079ff420-f2b5-4f29-a6de-9de5dfd76f64"> | <img width="627" alt="Screenshot 2024-02-21 at 11 37 10" src="https://github.com/WordPress/gutenberg/assets/846565/303cdd7c-7de3-4e8f-b314-7faf32d136ff"> |

We might try a stronger visual indicator than the hover state, but this felt a reasonable place to start.
